rules/hoc-transliteration.js

```diff
@@ -0,0 +1,102 @@
+( function ( $ ) {
+	'use strict';
+
+	var hoctransliteration = {
+		id: 'hoc-transliteration',
+		name: 'Ho Transliteration',
+		description: 'Ho transliteration input',
+		date: '2016-10-23',
+		URL: 'http://github.com/wikimedia/jquery.ime',
+		author: 'Mangu Purty',
+		license: 'GPLv3',
+		version: '1.0',
+		contextLength: 2,
+		maxKeyLength: 3,
+		patterns: [
+			[ 'a', '𑣁'],
+			[ 'b', '𑣗' ],
+			[ 'c', '𑣏' ],
+			[ 'd', '𑣑' ],
+			[ 'e', '𑣈' ],
+			[ '𑣒h', 't',  '𑣕' ], //for ot
+			[ 'g', '𑣋' ],
+			[ 'h', '𑣙' ],
+			[ 'i', '𑣆' ],
+			[ '([𑢪-𑢿]|[𑣊-𑣟])i', '$1𑣂' ], // for wi
```

Comment:
Looks like JavaScript cannot deal with it. In Chrome I get a "Range out of order in character class" error. In Firefox I get a "invalid range in character class" error. This probably happens because this script was recently-added to Unicode, and you cannot use these characters in ranges. I'm not sure how to fix this. Maybe @santhoshtr will have an idea.

Reply:
This still cannot work. The Warang Citi alphabet is special case, and cannot be used with regular expression ranges, like [𑢪-𑢿]. You need to implement a solution as described in https://mathiasbynens.be/notes/javascript-unicode , or do it without ranges.

Reply:
Okay, I get the point, I'll try doing it without ranges.